src/style.css

```diff
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: 900;
+}
+
+* {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  /* ? */
```

Comment:
what's that?

Reply:
I left question mark for myself to maybe think abt this part once again later. Forgot to remove, sry.